### PR TITLE
fix: upgrade gradle & libs and remove Apollo dependency

### PR DESCRIPTION
**Pull request by hamada147**
# Overview

Upgrade gradle & libs and remove Apollo dependency

### My PR contains...
* [x] No code changes (changes to documentation, CI, metadata, etc.)
* [ ] Bug fixes (non-breaking change which fixes an issue)
* [ ] Improvements (misc. changes to existing features)
* [ ] Features (non-breaking change which adds functionality)

### My changes...
* [ ] are breaking changes
* [x] are not breaking changes
* [ ] If yes to above: I have updated the documentation accordingly

### Documentation
* [x] My changes do not require a change to the project documentation
* [ ] My changes require a change to the project documentation
* [ ] If yes to above: I have updated the documentation accordingly

### Tests
* [x] My changes can not or do not need to be tested
* [ ] My changes can and should be tested by unit and/or integration tests
* [ ] If yes to above: I have added tests to cover my changes
* [ ] If yes to above: I have taken care to cover edge cases in my tests

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -100,32 +105,36 @@ allprojects {
...
-                }
-            }
+//            maven {
+//                this.name = "GitHubPackages"
+//                this.url = uri("https://maven.pkg.github.com/input-output-hk/atala-prism-didcomm-kmm")
+//                credentials {
+//                    this.username = System.getenv("ATALA_GITHUB_ACTOR")
+//                    this.password = System.getenv("ATALA_GITHUB_TOKEN")
+//                }
+//            }
```
Comment (cristianIOHK) [inline build.gradle.kts:137]: Do we need to keep this?

Comment (hamada147) [reply build.gradle.kts:137]: Nope, removing it.

**Review by cristianIOHK — APPROVED** [review]
